pkg/invoke/args.go

```diff
@@ -29,14 +30,18 @@ type inherited struct{}
 
 var inheritArgsFromEnv inherited
 
-func (_ *inherited) AsEnv() []string {
+func (i *inherited) AsEnv() []string {
+	// The parameter i reserved for future use
```

Comment:
It's not really reserved -- the point of `inherited` is that it stands in for "just use the calling process's environment".